frontend/src/components/elements/Audio/Audio.tsx

```diff
@@ -24,35 +24,22 @@ interface Props {
 }
 
 class Audio extends React.PureComponent<Props> {
-  private audioRef = React.createRef<HTMLAudioElement>()
-
-  public componentDidMount = (): void => {
-    this.updateTime()
-  }
-
-  public componentDidUpdate = (): void => {
-    this.updateTime()
-  }
-
-  private updateTime(): void {
-    if (this.audioRef.current) {
-      const startTime = this.props.element.get("startTime")
-      this.audioRef.current.currentTime = startTime
-    }
-  }
-
```

Comment:
Looks like you need to merge in the latest `develop`

Reply:
Merge conflict? The PR removes the code above, which implements the ability to set the start time for st.audio